includes/class-sensei-course.php

```diff
@@ -3367,8 +3341,21 @@ public static function get_courses_page_url() {
 	 */
 	public static function get_course_completed_page_url( $course_id ) {
 		$page_id = isset( Sensei()->settings->settings['course_completed_page'] ) ? intval( Sensei()->settings->settings['course_completed_page'] ) : 0;
-		$url     = $page_id ? get_permalink( $page_id ) : '';
-		$url     = $url && 0 < $course_id ? add_query_arg( 'course_id', $course_id, $url ) : '';
+
+		/**
+		 * Filter the course completed page ID.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @hook sensei_course_completed_page_id
+		 *
+		 * @param {int} $page_id Course completed page ID.
+		 * @return {int} Course completed page ID.
+		 */
+		$page_id = apply_filters( 'sensei_course_completed_page_id', $page_id );
```

Comment:
The only manual change in this file.
The rest is the result of `vendor/bin/phpcbf`